File: src/main/java/io/github/chaosdave34/kitpvp/companions/impl/AllayCompanion.java
package io.github.chaosdave34.kitpvp.companions.impl;

import io.github.chaosdave34.kitpvp.companions.Companion;
import net.minecraft.world.entity.EntityType;
import net.minecraft.world.entity.PathfinderMob;
import net.minecraft.world.entity.animal.allay.Allay;
import net.minecraft.world.level.Level;

public class AllayCompanion extends Companion {
    public AllayCompanion() {
        super("Allay", 5);
    }

    @Override
    public PathfinderMob createVanillaMob(Level world) {
        return new Allay(EntityType.ALLAY, world);
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/companions/impl/AllayCompanion.kt
package io.github.chaosdave34.kitpvp.companions.impl

import io.github.chaosdave34.kitpvp.companions.Companion
import net.minecraft.world.entity.EntityType
import net.minecraft.world.entity.PathfinderMob
import net.minecraft.world.entity.animal.allay.Allay
import net.minecraft.world.level.Level

class AllayCompanion : Companion("Allay", 5.0) {
    override fun createVanillaMob(world: Level): PathfinderMob {
        return Allay(EntityType.ALLAY, world)
    }
}